.github/workflows/docs-deploy.yml

```
@@ -95,7 +95,7 @@ jobs:
   deploy-github-pages:
     name: Deploy to GitHub Pages
     runs-on: ubuntu-latest
-    if: false  # Disabled by default - enable when GitHub Pages is configured
+    if: false # Disabled by default - enable when GitHub Pages is configured
```

actionlint (1.7.9)

condition "false" is always evaluated to false. remove the if: section

(if-cond)